live-examples/css-examples/animation/animation-delay.css

```diff
@@ -0,0 +1,53 @@
+#example-element {
+    background-color: #1766aa;
+    color: white;
+    margin: auto;
+    margin-left: 0;
+    border: 5px solid #333;
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+
```

Comment:
I'm not sure why there's a blank line here.

Reply:
Me neither! I'll remove it.